### PR TITLE
Add test for default expressions with File engine

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added test for #5666.

### CODE REVIEW

**Comment by qoega** [comment]
==273==ERROR: AddressSanitizer: allocator is out of memory trying to allocate 0x10000f bytes

File: tests/queries/0_stateless/01259_default_expressions_file_engine.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```
Comment (alexey-milovidov) [inline tests/queries/0_stateless/01259_default_expressions_file_engine.sh:1]: There is also `user_files_path` where it's allowed to read/write files.
You can create a file there also with `file` table function.

**Comment by alexey-milovidov** [comment]
Why close?

**Comment by nikitamikhaylov** [comment]
This test depends on config of the server. But I read again your comment and understand it.

**Pull request closed without merging**